go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/pelletier/go-toml v1.6.0
 	github.com/pkg/errors v0.8.1
 	github.com/rakyll/statik v0.1.6
+	github.com/regen-network/cosmos-proto v0.0.0-20200110230910-60122eba6250 // indirect
```

Comment:
Undo. Instead, update the `protoc-gen-gocosmos` target to install a specific version:

```make
protoc-gen-gocosmos:
  @(cd /tmp && github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@VERSION)
```